CRM/Civicase/Setup/UpdateMenuLinks.php

```diff
@@ -8,7 +10,7 @@ class CRM_Civicase_Setup_UpdateMenuLinks {
   /**
    * Manage case URL.
    */
-  const MANAGE_CASE_URL = 'civicrm/case/a/?case_type_category=cases#/case/list?cf=%7B"case_type_category":"cases"%7D';
```

Comment:
Apart from modifying this setup, which means that this will work for new installation, we need to add an upgrader for this to work on existing installations. See: https://github.com/compucorp/uk.co.compucorp.civicase/blob/29eb976a384416f0ec1825b07e481402ccd41092/CRM/Civicase/Upgrader/Steps/Step0009.php#L8